test/fixtures/readProjectFile.js

```diff
@@ -0,0 +1,12 @@
+const AdmZip = require('adm-zip');
+const fs = require('fs');
+
+module.exports = {
+    readAsBuffer: function (path) {
+        return new Buffer(fs.readFileSync(path));
+    },
+    readAsString: function (path) {
```

Comment:
These method names don't seem to make much sense to me... I know they are just in tests, but it might clarify things a bit to rename them.

maybe `readAsString` => `projectJsonFromZip` and
`readAsBuffer` => `readFileToBuffer`

Reply:
@gnarf, good point! How about `readAsString` => `extractProjectJson`?

Reply:
sounds even better than my suggetion :)